core/esmf-aspect-model-aas-generator/src/main/java/org/eclipse/esmf/aspectmodel/aspect/AASModelAspectGenerator.java

```diff
@@ -0,0 +1,57 @@
+package org.eclipse.esmf.aspectmodel.aspect;
+
+import org.apache.jena.rdf.model.Model;
+import org.eclipse.digitaltwin.aas4j.v3.model.Environment;
+import org.eclipse.esmf.aspectmodel.resolver.services.SammAspectMetaModelResourceResolver;
+import org.eclipse.esmf.aspectmodel.resolver.services.VersionedModel;
+import org.eclipse.esmf.aspectmodel.serializer.PrettyPrinter;
+import org.eclipse.esmf.aspectmodel.serializer.RdfModelCreatorVisitor;
+import org.eclipse.esmf.aspectmodel.vocabulary.Namespace;
+import org.eclipse.esmf.metamodel.Aspect;
+
+import java.io.*;
+import java.util.function.Function;
+
+public class AASModelAspectGenerator {
+
+    /**
+     * Generates an AASX archive file for a given Aspect and writes it to a given OutputStream provided by <code>nameMapper<code/>
```

Comment:
This comment seems wrong, it generates Aspect TTL?

Reply:
Yes, you are totally right..!

Thanks!